wfe/wfe.go

```diff
@@ -552,6 +552,78 @@ func (wfe *WebFrontEndImpl) verifyContacts(acct acme.Account) *acme.ProblemDetai
 	return nil
 }
 
+func (wfe *WebFrontEndImpl) UpdateAccount(
+	ctx context.Context,
+	logEvent *requestEvent,
+	response http.ResponseWriter,
+	request *http.Request) {
+	body, key, prob := wfe.verifyPOST(ctx, logEvent, request, wfe.lookupJWK)
+	if prob != nil {
+		wfe.sendError(prob, response)
+		return
+	}
+
+	// updateAcctReq is the ACME account information submitted by the client
+	var updateAcctReq struct {
+		Contact []string `json:"contact"`
+	}
+	err := json.Unmarshal(body, &updateAcctReq)
+	if err != nil {
+		wfe.sendError(
+			acme.MalformedProblem("Error unmarshaling body JSON"), response)
```

Comment:
nit: "body JSON" would probably be more descriptive if called "account update JSON body".